Models/NewCard.cs

```diff
@@ -131,35 +119,56 @@ public static class NewCard
+		}
+
+		public static void AddToPool(CardInfo card, List<AbilityIdentifier> abilityIdsParam = null,
+			EvolveIdentifier evolveId = null,
+			IceCubeIdentifier iceCubeId = null, TailIdentifier tailId = null)
+		{
 			NewCard.cards.Add(card);
+			handleIdentifiers(card, abilityIdsParam, evolveId, iceCubeId, tailId);
 
-			handleIdentifiers(card, abilityIds, evolveId, iceCubeId, tailId);
+			Plugin.Log.LogInfo($"Loaded custom card {card.name}!");
+		}
 
-			Plugin.Log.LogInfo($"Loaded custom card {name}!");
+		// TODO Implement a handler for custom appearanceBehaviour - in particular custom card backs
+		// TODO Change parameter order, and function setter call order to make more sense
+		// TODO Rename parameters to be more user friendly
+		public static void AddToPool(string name, string displayedName, int baseAttack, int baseHealth,
+			List<CardMetaCategory> metaCategories, CardComplexity cardComplexity, CardTemple temple,
+			string description = null, bool hideAttackAndHealth = false,
+			int bloodCost = 0, int bonesCost = 0, int energyCost = 0,
+			List<GemType> gemsCost = null, SpecialStatIcon specialStatIcon = SpecialStatIcon.None,
+			List<Tribe> tribes = null, List<Trait> traits = null, List<SpecialTriggeredAbility> specialAbilities = null,
+			List<Ability> abilities = null, List<AbilityIdentifier> abilityIdsParam = null,
+			EvolveParams evolveParams = null,
+			string defaultEvolutionName = null, TailParams tailParams = null, IceCubeParams iceCubeParams = null,
+			bool flipPortraitForStrafe = false, bool onePerDeck = false,
+			List<CardAppearanceBehaviour.Appearance> appearanceBehaviour = null, Texture2D defaultTex = null,
+			Texture2D altTex = null, Texture titleGraphic = null, Texture2D pixelTex = null,
+			GameObject animatedPortrait = null, List<Texture> decals = null, EvolveIdentifier evolveId = null,
+			IceCubeIdentifier iceCubeId = null, TailIdentifier tailId = null)
+		{
+			var createdCard = CreateCard(
+				name, displayedName, baseAttack, baseHealth, metaCategories, cardComplexity, temple, description,
+				hideAttackAndHealth, bloodCost, bonesCost, energyCost, gemsCost, specialStatIcon, tribes, traits,
+				specialAbilities, abilities, evolveParams, defaultEvolutionName, tailParams, iceCubeParams,
+				flipPortraitForStrafe, onePerDeck, appearanceBehaviour,
+				defaultTex, altTex, titleGraphic, pixelTex, animatedPortrait, decals
+			);
+
+			NewCard.AddToPool(createdCard, abilityIdsParam, evolveId, iceCubeId, tailId);
 		}
 
-		private static void handleIdentifiers(CardInfo card, List<AbilityIdentifier> abilityIds, EvolveIdentifier evolveId,
-		IceCubeIdentifier iceCubeId, TailIdentifier tailId)
+		private static void handleIdentifiers(CardInfo card, List<AbilityIdentifier> abilityIdsParam = null,
+			EvolveIdentifier evolveId = null, IceCubeIdentifier iceCubeId = null, TailIdentifier tailId = null)
 		{
 			// Handle AbilityIdentifier
-			List<AbilityIdentifier> toRemove = new List<AbilityIdentifier>();
```

Comment:
Same as above. This code block is not needed since IdentifierHandlers will add the ability to the cards.